server/channels/app/platform/web_hub.go

```diff
@@ -197,7 +197,16 @@ func (ps *PlatformService) InvalidateCacheForChannelPosts(channelID string) {
 
 func (ps *PlatformService) InvalidateCacheForUser(userID string) {
 	ps.Store.Channel().InvalidateAllChannelMembersForUser(userID)
-	ps.invalidateWebConnSessionCacheForUser(userID)
+	ps.ClearUserSessionCache(userID)
+
+	if ps.clusterIFace != nil {
+		msg := &model.ClusterMessage{
+			Event:    model.ClusterEventClearSessionCacheForUser,
+			SendType: model.ClusterSendBestEffort,
+			Data:     []byte(userID),
+		}
+		ps.clusterIFace.SendClusterMessage(msg)
+	}
```

Comment:
You don't need this anymore since `ClearUserSessionCache` already sends the cluster message.